src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -481,15 +481,32 @@ class ApplicationManagerImpl
    *
    * @return true on success, false if passthru is already in process
    */
+  // DEPRECATED
```

Comment:
Should now be a property `@deprecated` in the method description rather than a standalone comment. The same will need to be done in `application_manager.h`.

Reply:
Thanks, done.